src/display/editor/stamp.js

```diff
@@ -68,6 +71,22 @@ class StampEditor extends AnnotationEditor {
     );
   }
 
+  /** @inheritdoc */
+  static isHandlingMimeForPasting(mime) {
+    const { supportedTypes } = this;
+    return (
+      mime.startsWith("image/") &&
+      (supportedTypes.endsWith(mime) || supportedTypes.includes(mime + ","))
+    );
```

Comment:
This isn't a wonder of readability, in my opinion. Could we perhaps simplify things overall by modifying the existing getter, e.g. something along these lines?
```diff
diff --git a/src/display/editor/stamp.js b/src/display/editor/stamp.js
index d3d16fc32..00352c72f 100644
--- a/src/display/editor/stamp.js
+++ b/src/display/editor/stamp.js
@@ -67,17 +67,17 @@ class StampEditor extends AnnotationEditor {
     return shadow(
       this,
       "supportedTypes",
-      types.map(type => `image/${type}`).join(",")
+      types.map(type => `image/${type}`)
     );
   }

+  static get supportedTypesStr() {
+    return shadow(this, "supportedTypesStr", this.supportedTypes.join(","));
+  }
+
   /** @inheritdoc */
   static isHandlingMimeForPasting(mime) {
-    const { supportedTypes } = this;
-    return (
-      mime.startsWith("image/") &&
-      (supportedTypes.endsWith(mime) || supportedTypes.includes(mime + ","))
-    );
+    return this.supportedTypes.includes(mime);
   }

   /** @inheritdoc */
@@ -164,7 +164,7 @@ class StampEditor extends AnnotationEditor {
       document.body.append(input);
     }
     input.type = "file";
-    input.accept = StampEditor.supportedTypes;
+    input.accept = StampEditor.supportedTypesStr;
     this.#bitmapPromise = new Promise(resolve => {
       input.addEventListener("change", async () => {
         this.#bitmapPromise = null;
```